CircuitPython_RGBMatrix/simple_scroller.py

```diff
@@ -0,0 +1,56 @@
+# This example implements a simple two line scroller using
+# Adafruit_CircuitPython_Display_Text.  Each line has its own color
+# and it is possible to modify the example to use other fonts and non-standard
+# characters.
+
+import array
+
+from _pixelbuf import wheel
+import adafruit_display_text.label
+import board
+import displayio
+import framebufferio
+import rgbmatrix
+import terminalio
+displayio.release_displays()
+
+matrix = rgbmatrix.RGBMatrix(
+    width=64, height=32, bit_depth=1,
+    rgb_pins=[board.D6, board.D5, board.D9, board.D11, board.D10, board.D12],
+    addr_pins=[board.A5, board.A4, board.A3, board.A2],
+    clock_pin=board.D13, latch_pin=board.D0, output_enable_pin=board.D1)
+display = framebufferio.FramebufferDisplay(matrix, auto_refresh=False)
+
+line1 = adafruit_display_text.label.Label(
```

Comment:
add a few comments to this file pleze :) a lot of people are gonna rely/refer on it!

Reply:
okie, done